charts/mlflow/Chart.yaml

```diff
@@ -58,10 +58,10 @@ annotations:
       url: https://github.com/burakince/mlflow
   artifacthub.io/containsSecurityUpdates: "false"
   artifacthub.io/changes: |
-    - Update Mlflow image version to 2.1.1
+    - Update Mlflow image version to 2.6.0
   artifacthub.io/images: |
     - name: mlflow
-      image: burakince/mlflow:2.1.1
+      image: burakince/mlflow:2.6.0
```

Comment:
[`burakince/mlflow`](https://hub.docker.com/r/burakince/mlflow) has not released a version past `2.1.1`. There's an official MLFlow image at `ghcr.io/mlflow/mlflow:v2.6.0`, but it [doesn't](https://github.com/mlflow/mlflow/blob/master/docker/Dockerfile) [support](https://github.com/mlflow/mlflow/blob/abf4140/.github/workflows/push-images.yml#L36-L55) all features as `burakince/mlflow` (e.g. SQLite), which might require updating the chart as well. :thinking: